src/lib-components/BlockForm.vue

```diff
@@ -362,27 +369,272 @@ export default {
                 window.scrollTo(0, 0)
             }
         },
+        closeBlockForm() {
+            this.isClosed = !this.isClosed
+        },
     },
 }
 </script>
 
 <style lang="scss" scoped>
 .block-form {
+    border-radius: var(--rounded-slightly-all);
+    border-color: var(--color-primary-blue-03);
+    margin-bottom: var(--space-l);
+    max-width: $container-l-text + px;
+    padding: var(--space-xl);
+
+    @media #{$medium} {
+        max-width: 100%;
+        min-width: 600px;
+    }
+
+    @media #{$small} {
+        max-width: 100%;
+        min-width: 320px;
+    }
+
+    .formTitle {
+        @include step-1;
+        padding-bottom: 9px;
+        border-bottom: 1px solid var(--color-secondary-grey-01);
```

Comment:
`border-bottom: 1px solid var(--color-secondary-grey-02);`
i think we can delete line 399

Reply:
The padding or the border-bottom?

Reply:
both
- border goes on formTitleWrapper instead
- i'm not seeing any visual changes when removing the padding here, so i think it's unnecessary unless it's serving another purpose?